src/thinkspace/client/thinkspace-assets/styles/otbl-css/_helpers.spacing.scss

```diff
@@ -95,7 +95,7 @@
  */
 
 .spacing__margin-top--0 {
-  margin-top: 0;
+  margin-top: 0 !important;
```

Comment:
I've only added !important to the helper classes. Helper classes attributes need to take priority